www/content/attributes/hx-ws.md

```diff
@@ -41,11 +39,10 @@ The default reconnection interval is implemented with the full-jitter exponentia
 Own implementations can be provided by setting `htmx.config.wsReconnectDelay` to a function with
 `retryCount` as its only parameter.
 
-
 ### Test Web Sockets Server
 
-Htmx includes a WebSockets test server with many more examples of how to use Server Sent Events.  Download the htmx source code from GitHub and navigate to /test/servers/ws to experiment.
+Htmx includes a WebSockets test server with many more examples of how to use Server Sent Events. Download the htmx source code from GitHub and navigate to /test/ws-sse to experiment.
```

Comment:
Can you submit a version of this that includes just the factual changes, not all the formatting changes?